frontend/test/metabase/scenarios/native/reproductions/22822-explore-results-unsupported-databases.cy.spec.js

```diff
@@ -11,7 +11,7 @@ const questionDetails = {
   },
 };
 
-describe("issue 22822", { tags: "@external" }, () => {
+describe.skip("issue 22822", { tags: "@external" }, () => {
```

Comment:
This test uses MongoDB as an example for a DB not supporting nested queries. It will have to be rewritten to use some other condition.